### PR TITLE
[OYPD-241] Google Map API during installation.

### PR DESCRIPTION
**Pull request by Kerby70**
Jira https://propeople-us.atlassian.net/browse/OYPD-241
D.o https://www.drupal.org/node/2852320

Steps to review:
- [x] Manually install
- [x] See step "3rd party services"
- [x] At that step in field "Google Maps API Key" enter a string like https://github.com/ymcatwincities/openy/pull/131/files#diff-88f19bef09097e5fc1c1dcf92336950eR1 make some notable variation to allow verification.
- [x] Complete install
- [x] Go to /admin/config/services/geolocation
- [x] See the key entered is present in that form

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build441

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/441/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build441/profiles/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build448

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/448/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build448/profiles/openy/build/reports/behat

**Comment by alexschedrov** [comment]
1. We should set empty value here https://github.com/ymcatwincities/openy/pull/131/files#diff-88f19bef09097e5fc1c1dcf92336950eR1, because it's key from another project.

File: src/Form/ThirdPartyServicesForm.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    // Get Google Maps API Key container.
+    $config = \Drupal::service('config.factory')->get('geolocation.settings');
+
+    $form['#title'] = $this->t('Google Maps API Key');
+
+    $form['google_map_api_key'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Google Maps API key'),
+      '#default_value' => $config->get('google_map_api_key'),
+      '#description' => $this->t('For help see the <a href="https://developers.google.com/maps/documentation/javascript/get-api-key#key">Google Maps API documentation</a>.'),
```
Comment (alexschedrov) [inline src/Form/ThirdPartyServicesForm.php]: Let's set the same description as we have in contrib module:
```
Google Maps requires users to use a valid API key. Using the <a href="https://console.developers.google.com/apis" target="_blank">Google API Manager</a>, you can enable the <em>Google Maps JavaScript API</em>. That will create (or reuse) a <em>Browser key</em> which you can paste here.
```

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by Kerby70** [comment]
@Sanchiz New changes added, per last comment.

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build453

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/453/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build453/profiles/openy/build/reports/behat

**Comment by Kerby70** [comment]
@Sanchiz Latest commit removes the google maps api key from configuration.

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build454

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/454/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build454/profiles/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]

**Pull request merged**